tools/accuracy_checker/openvino/tools/accuracy_checker/launcher/opencv_launcher.py

```diff
@@ -130,6 +135,90 @@ def batch(self):
+            layer_shape = self._inputs_shapes[lstm_var]
+            input_data = infer_outputs[i].reshape(layer_shape) if infer_outputs else np.zeros(
+                layer_shape
+            )
+            feed_dict[lstm_var] = input_data
+        return feed_dict
+
+    def _data_to_blob(self, layer_shape, data, layout):  # pylint:disable=R0911,R0912
+        data_shape = np.shape(data)
+        if len(layer_shape) == 4:
+            if len(data_shape) == 5:
+                data = data[0]
+            if len(data_shape) == 3:
+                data = np.expand_dims(data, -1)
+            data_shape = np.shape(data)
+            if len(data_shape) < 4:
+                if len(np.squeeze(np.zeros(layer_shape))) == len(np.squeeze(np.zeros(data_shape))):
+                    return np.resize(data, layer_shape)
+            return np.transpose(data, layout) if layout is not None else data
+        if len(layer_shape) == 2:
+            if len(data_shape) == 1:
+                return np.transpose([data])
+            if len(data_shape) > 2:
+                if all(dim == 1 for dim in layer_shape) and all(dim == 1 for dim in data_shape):
+                    return np.resize(data, layer_shape)
+                if len(np.squeeze(np.zeros(layer_shape))) == len(np.squeeze(np.zeros(data_shape))):
+                    return np.resize(data, layer_shape)
+        if len(layer_shape) == 3 and len(data_shape) == 4:
+            return np.transpose(data, layout)[0] if layout is not None else data[0]
+        if len(layer_shape) == 1:
+            return np.resize(data, layer_shape)
+        if (len(data_shape) == 3) and (len(layer_shape) == 2) and (data_shape[0] == 1) and (
+                data_shape[1] == 1) and self.allow_reshape_input:
+            return data[0]
+        if layout is not None and len(layer_shape) == len(layout):
+            return np.transpose(data, layout)
+        if (
+                len(layer_shape) == 1 and len(data_shape) > 1 and
+                len(np.squeeze(np.zeros(layer_shape))) == len(np.squeeze(np.zeros(data_shape)))
+        ):
+            return np.resize(data, layer_shape)
+        return np.array(data)
+
+    def fit_to_input(self, data, layer_name, layout, precision, template=None):
+        layer_shape = tuple(self._inputs_shapes[layer_name])
+        data = self._data_to_blob(layer_shape, data, layout)
+        if precision:
+            data = data.astype(precision)
+        if data.shape != layer_shape:
+            if self.allow_reshape_input:
+                return data
+
+        return data.reshape(layer_shape)
+
+    def predict_sequential(self, inputs, metadata=None, **kwargs):
+        lstm_inputs_feed = self._fill_lstm_inputs()
+        results = []
+        for input_blobs in inputs:
+            input_blobs.update(lstm_inputs_feed)
+            for blob_name in input_blobs.keys():
+                input = input_blobs[blob_name].astype(np.float32)
```

Comment:
************* Module openvino.tools.accuracy_checker.launcher.opencv_launcher
openvino/tools/accuracy_checker/launcher/opencv_launcher.py:209: [W0622(redefined-builtin), OpenCVLauncher.predict_sequential] Redefining built-in 'input'